### ISSUE TITLE
Keyboard navigation goes into language picker when closed

### ISSUE THREAD

**Issue by erikkroes** [electron/electronjs.org#4581]
When navigating the page using a keyboard (pressing the tab key), the keyboard focus moved from interactive element to interactive element. When the language picker is closed, I would expected focus to go from the language picker "button" to the "the Equal Justice Initiative"-link. Instead, the focus goes inside the language picker and passes all available languages. The picker is visually hidden, but not for keyboard users. This can be a very confusing experience.

===

This finding of this issue has been recorded. It is part of my weekly accessibility stream on Twitch: twitch.tv/erikKroes
A recording should be up soon at youtube: https://www.youtube.com/channel/UCwrpGO76k2HyUQspl-rG2eQ/videos

**Comment by rafalguzniczak** [comment]
Hi, I resolved this problem in this [PR](https://github.com/electron/electronjs.org/pull/4638) ;) Please take a look 👍
